src/main/java/cooper/command/AddCommand.java

```diff
@@ -21,12 +24,22 @@ public AddCommand(String amount, boolean isInflow) {
 
     /**
      * The override function for executing the 'add' command, calls for 'add' and subsequently printing the status
-     * to the command line.
+     * to the command line if and only if the command is being accessed by an 'admin' level user.
+     * @param signInDetails
```

Comment:
oof these @params are throwing checkstyle error, think u gotta fill up the details of the params